docs/shared/migration/from-turborepo.md

```diff
@@ -0,0 +1,193 @@
+
+3. For each cacheable script, does it produce output in the file system?
+
+   Any folders identified will be added to the task's `outputs`. i.e. `"outputs": ["{projectRoot}/dist"]`
+
+This process adds `nx` to your `package.json` at the root of your workspace:
+
+```json {% fileName="package.json" %}
+{
+  "name": "my-workspace",
+  ...
+  "devDependencies": {
+    ...
+    "nx": "16.8.0"
+  }
+}
+```
+
+It also creates an `nx.json` based on the answers given during the setup process. This includes cacheable operations as well as some initial definition of the task pipeline.
+
+## Convert turbo.json into Nx Configuration
+
+Most of the settings in your `turbo.json` file can be converted directly into `nx.json` equivalents. The key configuration properties of `dependsOn`, `inputs` and `outputs` have a very similar syntax and can probably be copied over directly from the `turbo.json` `pipeline` into the `nx.json` `targetDefaults`.
+
+If you have project-specific tasks defined in the root `turbo.json` (i.e. `myreactapp#build`) or in project-level `turbo.json` files (i.e. `/packages/myreactapp/turbo.json`), those settings should go in the `nx` property of the project's `package.json` (i.e. `/packages/myreactapp/package.json`).
+
+[Specific configuration property conversions](#specific-configuration-property-conversions) are documented below.
+
+## Example
+
+Let's say you start with the following `turbo.json` file:
+
+```json {% fileName="/turbo.json" %}
+{
+  "$schema": "https://turbo.build/schema.json",
+  "pipeline": {
+    "build": {
+      "dependsOn": ["^build"],
+      "outputs": ["dist/**"]
+    },
+    "docs#build": {
+      "dependsOn": ["^build"],
+      "outputs": ["www/**"]
+    },
+    "test": {
+      "dependsOn": ["build"],
+      "outputs": []
+    },
+    "e2e": {
+      "dependsOn": ["build"],
+      "outputs": []
+    }
+  },
+  "globalDependencies": ["babel.config.json"]
+}
+```
+
+Creating the equivalent configuration with Nx yields the following files:
+
+```json {% fileName="/nx.json" %}
+{
+  "$schema": "./node_modules/nx/schemas/nx-schema.json",
+  "namedInputs": {
+    "sharedGlobals": ["babel.config.json"],
+    "default": ["{projectRoot}/**/*", "sharedGlobals"]
+  },
+  "targetDefaults": {
+    "build": {
+      "dependsOn": ["^build"],
+      "inputs": ["default"],
+      "outputs": ["{projectRoot}/dist"]
+    },
+    "test": {
+      "dependsOn": ["build"],
+      "inputs": ["default"]
+    },
+    "e2e": {
+      "dependsOn": ["build"],
+      "inputs": ["default"]
+    }
+  },
+  "tasksRunnerOptions": {
+    "default": {
+      "runner": "nx-cloud",
+      "options": {
+        "cacheableOperations": ["build", "e2e", "test"],
+        "accessToken": "..."
+      }
+    }
+  },
+  "affected": {
+    "defaultBase": "main"
+  }
+}
+```
+
+```jsonc {% fileName="/packages/docs/package.json" %}
+{
+  "name": "docs",
+  // etc...
+  "nx": {
+    "targets": {
+      "build": {
+        "dependsOn": ["^build"],
```

Comment:
This is the same as the default in `nx.json`, so I'd leave it out - it makes the config look longer than it needs to be